### DISCUSSION TITLE
Failed to install version 2.4.0 of "Local REST API".

### DISCUSSION THREAD

**Discussion by Qvickis** [coddingtonbear/obsidian-local-rest-api#117] [Q&A] [answered]
### Describe the bug

I get the error-message: `Failed to install plugin "Local REST API". `
Is the plugin tested with version 1.6.3 (that's the version I use on my Windows 11 PC). 

### To Reproduce

Everytime I try to install irrespective of restarts etc. 

### Operating System

Windows 11 Home, 22631.3737

### Browser

Chrome

### Browser Version

125.0.6422.142

**Comment by coddingtonbear** [comment, marked as answer]
The build for obsidian-local-rest-api was briefly broken last night when I did a release while on a train, only to discover that I needed to rebuild the project's `package-lock.json`, but didn't have reliable enough internet access to do so.  I've just released version 2.4.1, and It should be alright now, though; good luck!
